src/app.coffee

```diff
@@ -303,6 +380,34 @@ angular.module 'weaver',
 
 
 
+.directive('predicateTable', ($uibModal, $timeout) ->
+  {
+    restrict: 'E'
+    link: (scope, element) ->
+
+      predicate = scope.entity
+      dataset = scope.dataset
+
+      tableElement = element[0]
+
+      containerDiv = document.createElement('div')
+      containerDiv.innerHTML = '<br/><br/><br/><br/>'
+      element[0].appendChild(containerDiv)
+  }
+)
+
+
+
+
+
+
+
+
+
+
+
+
+
 .directive('objectTable', (ObjectTableService, $uibModal, $timeout) ->
```

Comment:
1 blank line: ok
2 blank lines: eeehhh if really needed
3 blank lines: this is getting stupid
This amount of blank lines: Stop spilling coke on your keyboard, your enter key is stuck.